### DISCUSSION TITLE
brandLogo method is not exist

### DISCUSSION THREAD

**Discussion by bismaashfaq53** [filamentphp/filament#9186] [Help]
Why logo is not seen when i give a path its get error brandLogo() method is not exist

**Comment by sinnbeck** [comment]
Please show your code

**Comment by leandrocfe** [comment]
What is the Filament version?

**Comment by Jericho-Tsalis** [comment]
I have the same issue, I'm using:
filament version 3.0-stable.
Laravel Version 10

BadMethodCallException
Method Filament\Panel::brandLogo does not exist.

below: app/Providers/Filament/AdminPanelProvider.php


[AdminPanelProvider.php.txt](https://github.com/filamentphp/filament/files/13413048/AdminPanelProvider.php.txt)

**Reply by leandrocfe** [reply]
add ^3.0 in the composer.json
`"filament/filament": "^3.0"`
run `composer update`

**Reply by jhoanborges** [reply]
fixed with this. I had 3.0-stable

**Reply by ShaungBhone** [reply]
Need to add specific version!
```
"filament/filament": "^3.1.22"
```
delete composer.lock file and then `composer update`.
